packages/fhir-sanitize-param/yarn.lock

```diff
@@ -2,677 +2,19 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5":
```

Comment:
I think we can just put this file back. Was probably removed by accident with npm somehow?

Reply:
I think it was because I don't remember messing with that file at all